plugins/api/src/replica_close.cpp

```diff
@@ -176,13 +176,15 @@ namespace
             return ec;
         }
 
+        const auto cond_input = irods::experimental::key_value_proxy(_l1desc.dataObjInp->condInput);
+
         // Send an empty JSON structure if we do not want to trigger file_modified
         // as this is the way the RST interface determines whether it should trigger.
-        const auto file_modified = _send_notifications ? irods::to_json(&_l1desc.dataObjInp->condInput) : json{{}};
+        const auto file_modified = _send_notifications ? irods::to_json(cond_input.get()) : json{{}};
```

Comment:
Just needed an empty JSON struct. I think I've seen this sometimes get turned into a `json::array` instead of just a `json::basic_json`. I can change to whatever is most appropriate.

Reply:
i think the inner curly braces are confusing. if you used `json{}` and everything is sane, i think that would be better.

Reply:
Sure thing.